types/shipit-cli/index.d.ts

```diff
@@ -39,12 +43,36 @@ export interface Task {
 export function blTask(name: string, depsOrFn: string[] | EmptyCallback, fn?: () => void): any;
 export function emit(name: string): any;
 export function initConfig(config: {}): typeof shipit;
-export function local(command: string, options?: child_process.ExecOptions, callback?: (error: Error, stdout: string, stderr: string) => void): PromiseLike<ShipitLocal>;
+export function local(
+    command: string,
+    options?: child_process.ExecOptions,
+    callback?: (error: Error, stdout: string, stderr: string) => void,
+): PromiseLike<ShipitLocal>;
 export function log(log: any): void;
 export function log(...log: any[]): void;
 export function on(name: string, callback: (e: any) => void): any;
-export function remote(command: string, options?: child_process.ExecOptions, callback?: (error: Error, stdout: string, stderr: string) => void): PromiseLike<ShipitLocal>;
-export function remoteCopy(src: string, dest: string, options?: child_process.ExecOptions, callback?: (error: Error, stdout: string, stderr: string) => void): PromiseLike<ShipitLocal>;
+export function remote(
+    command: string,
+    options?: child_process.ExecOptions,
+    callback?: (error: Error, stdout: string, stderr: string) => void,
+): PromiseLike<ShipitLocal>;
+
+/**
+ * @deprecated Use `copyToRemote` or `copyFromRemote` instead
+ */
+export function remoteCopy(
+    src: string,
+    dest: string,
+    options?: child_process.ExecOptions,
+    callback?: (error: Error, stdout: string, stderr: string) => void,
+): PromiseLike<ShipitLocal>;
+export function copyToRemote(src: string, dest: string, options?: child_process.ExecOptions): PromiseLike<ShipitLocal>;
+export function copyFromRemote(
+    src: string,
+    dest: string,
+    options?: child_process.ExecOptions,
+): PromiseLike<ShipitLocal>;
```

Comment:
This is the main change I propose in this file ( other diffs are made by prettier )